staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -168,6 +172,12 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 		if callErr, ok := err.(*webhookutil.ErrCallingWebhook); ok {
 			if ignoreClientCallFailures {
 				klog.Warningf("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
+
+				key := MutationAuditAnnotationPrefix + MutationAuditAnnotationFailedOpenKeySuffix
```

Comment:
Audit annotations are a string[string] map. Using a constant key like this means that if multiple webhooks fail open, only the first one will get recorded in an audit annotation.